caluma/caluma_workflow/migrations/0026_auto_20200901_1435.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.15 on 2020-09-01 14:35
+
+import caluma.caluma_core.models
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("caluma_workflow", "0025_auto_20200901_1415")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="case",
+            name="status",
+            field=caluma.caluma_core.models.ChoicesCharField(
+                choices=[
+                    ("running", "Case is running and work items need to be completed."),
+                    ("completed", "Case is done."),
+                    ("canceled", "Case is canceled."),
```

Comment:
Am I missing something or should we also provide an migration for moving existing status from "cancelled" to "canceled"?
The current migration just changes the value of the choices.

Reply:
The value did not change but the description to said value. Check out the changes in the models.